elementary/monitor/dbt_project/models/enriched_exposures/enriched_exposures.sql

```diff
@@ -5,9 +5,23 @@ config(
   unique_key='unique_id',
 )
 }}
+{% set dbt_exposures_relation = ref('elementary', 'dbt_exposures') %}
+{% set dbt_exposures_columns = adapter.get_columns_in_relation(dbt_exposures_relation) %}
+{% set raw_queries_column_exists = elementary.column_exists_in_relation(dbt_exposures_relation, 'raw_queries') %}
+{% set query %}
+    select
+    {% if raw_queries_column_exists %}
+        *
+    {% else %}
+        {% for column in dbt_exposures_columns %}
```

Comment:
You know the schema here so I'd just expand it manually instead of doing the get_columns_in_relation which is probably slower as it runs describe on the warehouse